AIPscan/Aggregator/templates/storage_service.html

```diff
@@ -67,6 +76,13 @@
   $('#newfetchjob').on('click', function () {
     new_fetch_job({{ storage_service.id }});
     });
+  $('.stats').hide();
+  $('.fa-info-circle').click(function() {
+    if($('#stats-'+this.id).is(':visible'))
+        $('#stats-'+this.id).hide();
+    else
+        $('#stats-'+this.id).show();    
+    });
```

Comment:
Succinct, readable, and effective - nice job!

One minor nit/linting issue is that it'd be great to have opening and closing curly braces aligned - these are fairly small closures so not an immediate practical need maybe, but having them properly aligned really helps with readability when code grows. In practical terms I think it would just mean moving the closing `});`s here two spaces to the left.

Reply:
This is the sort of thing that we will ideally eventually have a JS linter to help spot (and potentially even auto-correct for us, like `black` does for our Python). And it's absolutely a nit, so I'll leave it up to you to decide if it's worth the time.

Reply:
I'll take any suggestions on improving the readability of Javascript / JQuery, lol. I've moved the bracket as suggested. Have we shortlisted JS Linters yet? I'll gladly install one.